documentation/book/assembly-access-configuration-kafka-mirror-maker.adoc

```diff
@@ -0,0 +1,18 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='assembly-access-configuration-kafka-mirror-maker-{context}']
+= Configuring Kafka Mirror Maker access control
+
+Configure Kafka Mirror Maker access control using supported authentication mechanisms:
```

Comment:
I think the intro here is a bit confusing. You list 3 things:
* TLS client authentication
* SASL SCRAM-SHA-512
* SASL PLAIN

So I expected 3 chapters about each of these. But in reality you talk about encryption not entioned in the list and only about TLS client authentication and SASL SCRAM. 

I think it would be good to mention the encryption here more visibly. We should also add SASL PLAIN to the name of the chapter about SCRAM-SHA-512, since it is actually covered there.

Reply:
Thanks. I've updated to match the procedures.